frontend/imports/utils/conversion.js

```diff
@@ -1,15 +1,36 @@
-import { Dapple, web3 } from 'meteor/makerotc:dapple';
+import { Dapple } from 'meteor/makerotc:dapple';
 import { BigNumber } from 'meteor/ethereum:web3';
 
-export default function convertToWei(amount, token) {
-  const tokenSpecs = Dapple.getTokenSpecs(token);
-  const precision = tokenSpecs !== undefined ? tokenSpecs.precision : 18;
-  if (precision === 18) {
-    return web3.toWei(amount);
+export function convertToAbsolute(amount, token) {
```

Comment:
also is `web3.toWei` no longer necessary? was it only being used because it also uses 18 digits of precision?